src/property.ts

```diff
@@ -90,6 +94,11 @@ export const NESTED_ENCODING_PROPERTIES: NestedEncodingProperty[] = [
     property: Property.SCALE_TYPE,
     parent: 'scale',
     child: 'type'
+  },
+  {
+    property: Property.SCALE_ZERO,
```

Comment:
Btw, rename `NESTED_ENCODING_PROPERTY_INDEX` to `NESTED_ENCODING_PROPERTY_PARENT_INDEX`

because the key is parent.  

(While `NESTED_ENCODING_INDEX` below, has the property itself as key.)